apistructs/file_records.go

```diff
@@ -89,5 +89,10 @@ type GetTestFileRecordResponse struct {
 
 type ListTestFileRecordsResponse struct {
 	Header
-	Data []TestFileRecord
+	Data *ListTestFileRecordsResponseData
```

Comment:
struct changes will break api compatibility, how do you deal with it?

Reply:
It is not backward-compatible, as not designed well at the beginning. We may version it in the future. Updated API docs here: https://erda.cloud/erda/workBench/projects/387/apps/5880/apiDesign

Reply:
so must frontend change their code at same time?